hi-ml-histopathology/docs/README.md

```diff
@@ -0,0 +1,45 @@
+# Using the histopathology models
+
+## Onboarding to Azure
+
+Prio 1, to be completed later
+
+## Creating datasets
+
+* [How to get PANDA (whole slides) and upload it to Azure](panda_dataset.md)
+* [How to get TCGA-Crck (tiles) and upload it to Azure](tcga-crck_dataset.md)
+
+## Training a model
+
+* Prio 0: [PANDA DeepMil on whole slides with ImageNet encoder](panda_model.md)
+  * How to invoke training for the existing model on PANDA data
+  * Where to find results and how to interpret them
+  * What results are expected (accuracy)
+  * If people want to re-use that model on their data:
+    * What is the expected data format?
+    * Which parameters need to be tuned?
+* Prio 1: [TCGA-Crck DeepMil on tile dataset with ImageNet encoder](tcga-crck_model.md)
```

Comment:
Prio 0/1 are internal tags -do we want to merge this in main or keep it on a dev_branch until completed?

Reply:
I agree it'd make sense to host the incomplete docs on a separate branch.